networks/typical/main.tf

```diff
@@ -13,51 +13,69 @@ provider "docker" {
 
 locals {
   number_of_nodes = var.number_of_nodes
-  node_indices    = range(local.number_of_nodes)
+  node_indices = range(local.number_of_nodes)
   more_args = join(" ", [
-    "--allow-insecure-unlock" # since 1.9.7 upgrade
+    "--allow-insecure-unlock",
+    # since 1.9.7 upgrade,
+    "--revertreason"
   ])
 }
 
 module "helper" {
   source = "../_modules/docker-helper"
 
-  consensus       = var.consensus
+  consensus = var.consensus
   number_of_nodes = local.number_of_nodes
   geth = {
     container = {
-      image   = var.quorum_docker_image
-      port    = { raft = 50400, p2p = 21000, http = 8545, ws = -1 }
+      image = var.quorum_docker_image
+      port = {
+        raft = 50400,
+        p2p = 21000,
+        http = 8545,
+        ws = -1
+      }
       graphql = true
     }
     host = {
-      port = { http_start = 22000, ws_start = -1 }
+      port = {
+        http_start = 22000,
+        ws_start = -1
+      }
     }
   }
   tessera = {
     container = {
       image = var.tessera_docker_image
-      port  = { thirdparty = 9080, p2p = 9000, q2t = 9081, q2t = 9081 }
+      port = {
+        thirdparty = 9080,
+        p2p = 9000,
+        q2t = 9081,
+        q2t = 9081
+      }
     }
     host = {
-      port = { thirdparty_start = 9080, q2t_start = 49081 }
+      port = {
+        thirdparty_start = 9080,
+        q2t_start = 49081
+      }
     }
   }
 }
 
 module "network" {
   source = "../_modules/ignite"
 
-  concensus            = module.helper.consensus
```

Comment:
is this a kind of auto-format that does all these modifications?

Reply:
also concensus instead of consensus

Reply:
yes, is an auto format. I need to match the format of intellij with terraform fmt. I will push a new branch to fix that.

@dolanor good point. I will fix that typo too.

Reply:
fixed on https://github.com/ConsenSys/quorum-acceptance-tests/pull/121